binary.json

```diff
@@ -46,8 +46,13 @@
          "expected": 31
       },
       {
-         "description": "invalid binary numbers raise an error",
-         "binary": ["012", "10nope", "nope10", "10nope10", "001 nope", "2"],
+         "description": "numbers other than one and zero raise an error",
+         "binary": ["012", "2"],
+         "expected": -1
+      },
+      {
+         "description": "containing letters raises an error",
+         "binary": ["10nope", "nope10", "10nope10", "001 nope"],
```

Comment:
Binary should not be an array.
See my other comment above about splitting arrays into individual tests.